website/docs/docs/dbt-versions/core-versions.md

```diff
@@ -99,24 +99,27 @@ We are committed to avoiding breaking changes in minor versions for end users of
 
 When you use dbt, you use a combination of `dbt-core` and an adapter plugin specific to your database. You can see the current list in [Supported Data Platforms](/docs/supported-data-platforms). Both `dbt-core` and dbt adapter plugins follow semantic versioning.
 
-`dbt-core` and adapter plugins coordinate new features and behind-the-scenes changes in minor releases. When fixing bugs, sooner is better, so patch versions are released independently for `dbt-core` and plugins.
+`dbt-core` and adapter plugins coordinate new features and behind-the-scenes changes via the `dbt-adapters` interface. New features in adapters, and that `dbt-core` will leverage, are defined in `dbt-adapters`. These features are opt-in so they only impact adapters that explicitly implement them. This allows us to independently release adapters, `dbt-adapters`, and `dbt-core` without creating a broken experience for users.
```

Comment:
this is great but i struggle to understand first time reading so tried to simplify
```suggestion
`dbt-core` and adapter plugins use the `dbt-adapters` interface to coordinate new features and behind-the-scenes changes. `dbt-adapters` defines new features in adapters that `dbt-core` uses. These features are opt-in, meaning they only impact adapters that explicitly implement them. This allows us to independently release adapters, `dbt-adapters`, and `dbt-core` without creating a broken experience for users.
```